src/Microsoft.Windows.CsWin32/Generator.cs

```diff
@@ -1184,6 +1205,12 @@ internal void RequestInteropType(TypeDefinitionHandle typeDefHandle)
                             new SyntaxAnnotation(NamespaceContainerAnnotation, shortNamespace));
```

Comment:
tip: This could be written without an `if` block like this:

```cs
                    this.needsWinRTCustomMarshaler |= typeDeclaration.DescendantNodes().OfType<AttributeSyntax>()
                        .Any(a => a.Name.ToString() == "MarshalAs" && a.ToString().Contains(WinRTCustomMarshalerFullName));
```

Reply:
Fixed.